src/WebJobs.Script.WebHost/Program.cs

```diff
@@ -79,6 +77,13 @@ public static IWebHostBuilder CreateWebHostBuilder(string[] args = null)
 
                     loggingBuilder.AddDefaultWebJobsFilters();
                     loggingBuilder.AddWebJobsSystem<WebHostSystemLoggerProvider>();
+
+                    if (!FeatureFlags.IsEnabled(ScriptConstants.FeatureFlagDisableDiagnosticEventLogging))
```

Comment:
I see we are doing explicit disable via feature flag. Should we be doing explicit enable? Not all customers use customer startup classes. So Why not those small number of customers explicitly enable this feature flag as needed, so that no cold start impact for everyone.

Reply:
Couple responses to set up the explanation:
- this diagnostic event logging already exists and is used to surface any errors to the customer directly via the portal (the portal can ask us if there are any events, then pops them up)
- today, this is only supported in the ScriptHost. It's set up here: https://github.com/Azure/azure-functions-host/blob/dev/src/WebJobs.Script.WebHost/WebScriptHostBuilderExtension.cs#L94-L99
- but I need this in the WebHost -- specifically to surface issues where the ScriptHost cannot be built b/c of exceptions
- this won't affect cold start -- everything will already be constructed and warmed up by the time this happens

All that being said -- this feature flag check won't even work today b/c of placeholder mode. We'll need this to be *always* registered, then filter it away if it's explicitly disabled. I need to refactor all of this a bit. I'm working on that now.